docs/preview/features/servicebus-messsage-pump.md

```diff
@@ -0,0 +1,179 @@
+﻿---
+title: Testing Azure Service Bus message handling
+layout: default
+---
+
+# Testing Azure Service Bus message handling
+
+## Installation
+The following functionality is available when installing this package:
+
+```shell
+PM> Install-Package -Name Arcus.Testing.Messaging.Pumps.ServiceBus
+```
+
+## Test Azure Service Bus message pump
+As an addition on the [Arcus Azure Service Bus message pump](https://messaging.arcus-azure.net/Features/message-handling/service-bus), we have provided a test version of the message pump to verify your custom Azure Service Bus message handler implementations.
+These hander implementations can be tested separately, and could be tested by interacting with the message router directly, but simulating messages like it would be from Azure Service Bus itself is a bit ticker.
```

Comment:
What do you mean with 'ticker' ?  Or do you mean 'trickier' ?

Reply:
Hihi 😄 , yeah, meant 'trickier', like 'harder to do'.

Reply:
https://github.com/arcus-azure/arcus.testing/pull/68/commits/072af31cae2b81a61b4771c28b0538f9673e52ca